CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # edv-client ChangeLog
 
+## 8.1.0 - 2021-04-TBD
+
+### Changed
+- **BREAKING**: Use (@digitalbazaar/minimal-cipher@3.0)[https://github.com/digitalbazaar/minimal-cipher/blob/master/CHANGELOG.md]
```

Comment:
Please add subpoint: Now uses `X25519KeyAgreementKey2020` instead of `X25519KeyAgreementKey2020`.

Reply:
Updated.